lib/beaker/host.rb

```diff
@@ -42,33 +43,33 @@ def [](k)
 
     def self.create name, host_hash, options
       case host_hash['platform']
-      when /windows/
-        cygwin = host_hash['is_cygwin']
-        if cygwin.nil? or cygwin == true
-          Windows::Host.new name, host_hash, options
+        when /windows/
```

Comment:
in beaker, we've always standardized on the `when` clause going at the same level of the `case` in a case statement.